script-reference/types.rst

```diff
@@ -1211,7 +1213,8 @@ Vectorized operations
 
 Vectors of integral types (:zeek:type:`int` or :zeek:type:`count`) support the pre-increment
 (``++``) and pre-decrement operators (``--``), which will increment or
-decrement each element in the vector.
+decrement each element in the vector. It also support the bitwise right shift (``<<``) 
```

Comment:
```suggestion
decrement each element in the vector. They also support the bitwise right shift (``<<``) 
```

Reply:
Now that https://github.com/zeek/zeek/pull/2309 was merged. This section can be removed since bit-shifting vectors returns a deprecation warning.